### PR TITLE
Update Time to match Rep Dom Time Zone

### PR DESCRIPTION
**Pull request by lecollabdul**
This fixed reported issue number 1:

https://github.com/ricardov03/que-toca/issues/1

Raired with @kajahno on this one.

### CODE REVIEW

File: src/components/Clock.vue [hunk 1]
```diff
@@ -79,18 +79,21 @@ export default {
       }
     },
     showTime() {
-      const date = new Date()
+      const timeZone = -4 // GTM -4:00
+
+      const d = new Date()
+      const utc = d.getTime() + (d.getTimezoneOffset() * 60000)
+      // getTime -> milliseconds since epoch (absolute time)
```
Comment (ricardov03) [inline src/components/Clock.vue:86]: Thanks for share the timezone match formula. This could be helpful for future time-sensitive projects.

**Review by ricardov03 — APPROVED** [review]
Tested it on my local environment and the time it's showing right. I don't know how to test an alternative timezone locally but let's talk with @kajahno to check if the time is fixed after the merge.

**Comment by kajahno** [comment]
Looks good, thanks. This is one of those things that can be easily mocked by a unit test but not so easy via fiddling with the browser, for example:

```javascript
// testcase: 20210101T030000 GMT+0 === 20201231T230000 GMT-4
timeZone = -4
val = "Fri, 01 Jan 2021 03:00:00 GMT"
expected = "Thu, 31 Dec 2020 23:00:00 GMT"

d = new Date(val)
utc = d.getTime() + (d.getTimezoneOffset() * 60000)
date = new Date(utc + (3600000 * timeZone))

date.toGMTString() === expected // true
```

**Comment by ricardov03** [comment]
We are not using any testing library yet on this application. I would like to implement E2E testing with Cypress. I think this kind of testing could be accomplished with Jest. Any other recommendations?

**Comment by kajahno** [comment]
thanks that sounds great!

**Pull request merged**